r/rsparkling/R/package.R

```diff
@@ -1,13 +1,24 @@
-
 #' @import sparklyr h2o utils
 NULL
 
 # define required spark packages
 spark_dependencies <- function(spark_version, scala_version, ...) {
+  sw_version = getOption("rsparkling.sparklingwater.version", default = "1.6.7")
```

Comment:
@jakubhava Are you thinking of something like this:

`  sw_version = getOption("rsparkling.sparklingwater.version", default = NULL)
  if(is.null(sw_version)){
    stop("Sparkling Water version is not set. Please choose a correct version")
  }`